inbm/dispatcher-agent/dispatcher/dispatcher_broker.py

```diff
@@ -35,31 +36,72 @@ def start(self, tls: bool) -> None:  # pragma: no cover
         self.mqttc.start()
         self._is_started = True
 
-    def send_result(self, message: str, id: str = "") -> None:  # pragma: no cover
-        """Sends event messages to local MQTT channel        
+    def send_update(self, message: str, job_id: Optional[str]) -> None:
+        """Sends node update to local MQTT 'UPDATE' channel to be published
+        to the cloudadapter where it will be sent as a reques to INBS (service in UDM)       
 
         Raises ValueError if id contains a slash
 
         @param message: message to be published to cloud
-        @param id: if not "", publish to RESPONSE_CHANNEL/id instead of RESPONSE_CHANNEL
+        @param job_id: Job ID used to track the request in both UDM and TC
         """
-        if id:
-            extra_log = f" with id {id}"
+        if job_id is None:
+            raise ValueError("job_id cannot be None")
+        new_message = job_id + ":" + message
+        logger.debug(f"Sending node update for job={job_id} to {UPDATE_CHANNEL} with message: {new_message}")
+        self.mqtt_publish(topic=UPDATE_CHANNEL, payload=job_id + ":" + message)
+     
+    def _check_db_for_started_job(self) -> Schedule:
+        sqliteMgr = SqliteManager()
+        schedule = sqliteMgr.get_any_started_schedule()
+         
+        if schedule:
+            logger.debug("Found started schedule in DB")
+            s = Schedule(request_id=schedule.request_id,
+                         job_id=schedule.job_id, 
+                         task_id=schedule.task_id,
+                         schedule_id=schedule.schedule_id)
+            # Change status to COMPLETED
+            sqliteMgr.update_status(s, COMPLETED)
+        
+        del sqliteMgr
```

Comment:
instead of del sqliteMgr, wrap with a 'with' statement -- with SqliteManager() as sqliteMgr: (rest of code)